spec/unit/reservation_list_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe "Reservation index list", type: :feature, js: true do
+
+  let(:user) { FactoryGirl.create(:user) }
+
+  it "should get reservation lists from proper day" do
+    FactoryGirl.create(:reservation, from: '10', to: '11')
+    FactoryGirl.create(:reservation, from: '12', to: '13')
+    FactoryGirl.create(:reservation, from: '14', to: '15')
+    Reservation.create(reserved_at: (Time.now + 86400), from: '10', to: '11')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.